src/components/Layout/HomeContent.js

```diff
@@ -160,7 +164,11 @@ export function HomeContent() {
 
         <Section background="left-card">
           <Center>
-            <Header>컴포넌트를 사용하여 사용자 인터페이스 만들기</Header>
+            <Header>
+              컴포넌트를 사용하여
+              <Br />
+              사용자 인터페이스 만들기
```

Comment:
```suggestion
 컴포넌트를 사용하여 <Br /> 사용자 인터페이스 만들기
```

이렇게 한 줄로 쓰면 포맷팅도 깔끔하고 띄어쓰기도 정상 작동합니다

Reply:
이건 이렇게 수정한 뒤 푸시하면 자동으로 원래대로 바뀌길래 강제로도 푸쉬해보았습니다만, eslint와 prettier 세팅 때문에 거절되어 변경이 불가능한 것 같습니다 ㅠㅠ 😭